tests/helpers.py

```diff
@@ -248,6 +277,20 @@ def create_secret(interface_type):
 
 
 def create_ceph_block_pool(pool_name=None):
+    """
+    Returns defaul CephBlockPool
```

Comment:
This emphasize what I mean in above comment.
Function name 'create...' but you are not creating. 
So using a function 'return_default_...' and using in with additional parameter in the original function, should be simpler imo

Reply:
Based on my testing this function call is not needed anymore once the storageclass function is redone.